draft-thomson-webpush-protocol.xml

```diff
@@ -364,7 +407,71 @@ Location: https://push.example.net/d/qDIYHNcfAIPP_5ITvURr-d6BGtYnTRnk
         Push services MUST NOT generate a 413 status code in responses to an
         entity body that is 4k (4096 bytes) or less in size.
       </t>
+      <section anchor="request_receipt" title="Receiving Push Message Receipts">
```

Comment:
This could probably be split into two: "Requesting Push Message Receipts" and "Receiving Push Message Receipts"